prompting/validators/reward/blacklist.py

```diff
@@ -32,7 +32,8 @@
 
 @dataclass
 class BlacklistRewardEvent(BaseRewardEvent):
-    is_filter_model: bool = True
+    matched_ngram: str = None
+    significance_score: float = None
```

Comment:
Lets keep matched_ngram / significance_score removed from the reward event, cause we are not using them rn, and they will just be logging Nans to wandb